packages/babel-plugin-transform-vite-meta-glob/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [1.0.1](https://github.com/OpenSourceRaidGuild/babel-vite/compare/babel-plugin-transform-vite-meta-glob@1.0.0...babel-plugin-transform-vite-meta-glob@1.0.1) (2021-03-09)
+
+
+### Bug Fixes
+
+* added license ([8c01a6a](https://github.com/OpenSourceRaidGuild/babel-vite/commit/8c01a6a5e17a4bacc9f29f4282ef1fcaad85bad8))
+
+
+
+
+
+# 1.0.0 (2021-03-09)
```

Comment:
should this be `##` like `## 1.0.1....` above?

Reply:
This first version wasn't generated with conventional commits. I just did `npx lerna version --repo-version 1.0.0`